website/guides/managing-kubernetes-with-terraform.html.md

```diff
@@ -0,0 +1,585 @@
+---
+layout: "guides"
+page_title: "Managing Kubernetes with Terraform - Guides"
+sidebar_current: "guides-managing-kubernetes-with-terraform"
+description: |-
+  This guide focuses on scheduling Kubernetes resources like Pods,
+  Replication Controllers, Services etc. on top of a properly configured
+  and running Kubernetes cluster.
+---
+
+# Managing Kubernetes with Terraform
+
+## Kubernetes
+
+[Kubernetes](https://kubernetes.io/) is an open-source workload scheduler
+with focus on containerized applications.
+
+There are at least 2 steps involved in scheduling your first container
+on a K8s cluster. You need the K8S cluster with all its components
+running _somewhere_ and then schedule the K8S resources, like Pods,
+Replication Controllers, Services etc.
+
+This guide focuses mainly on the latter part and expects you to have
+a properly configured & running Kubernetes cluster.
+
+The guide also expects you to run the cluster on a cloud provider
+where K8S can automatically provision a load balancer.
+
+## Why Terraform?
+
+While you could use `kubectl` or similar CLI-based tools mapped to API calls
+to manage all K8S resources described in YAML files,
+orchestration with Terraform presents a few benefits.
+
+ - [HCL](https://www.terraform.io/docs/configuration/syntax.html) - same language
+    for both lower layer with underlying infrastructure (compute) and scheduling layer
+ - drift detection - `terraform plan` will always present you the difference
+    between reality at a given time and config you intend to apply.
+ - full lifecycle management - Terraform doesn't just initially create resources,
+    but offers a single command for creation and deletion of tracked resources
```

Comment:
is update supported too? In which case this might be better to mention "managing the lifecycle"?